src/Kmd.Logic.Gateway.Automation/PreValidation/GatewayValidationResult.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Kmd.Logic.Gateway.Automation
+{
+   public class GatewayValidationResult
+    {
+        public bool IsError { get; set; }
+
+        public IEnumerable<PublishResult> PublishResults { get; set; }
```

Comment:
Validation result contains PublishResults?

Reply:
Renamed to ValidationResult